test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -48,61 +49,61 @@ let paused = false;
 }
 
 {
-  const file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
-  file3.length = 0;
-  file3.on('data', function(data) {
+  const file = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+  file.length = 0;
+  file.on('data', function(data) {
```

Comment:
Do we have any preferences whether to use lambdas or good old function declarations when they are interchangeable?